controllers/antreainstall_controller.go

```diff
@@ -216,6 +142,122 @@ func (r *AntreaInstallReconciler) Reconcile(request ctrl.Request) (ctrl.Result,
 			}
 		}
 	}
+	return reconcile.Result{}, nil
+}
+
+func (k8s *AdaptorK8s) Reconcile(r *AntreaInstallReconciler, request ctrl.Request) (ctrl.Result, error) {
```

Comment:
For `func (k8s *AdaptorK8s) Reconcile` and `func (oc *AdaptorOc) Reconcile`, there are similar logic for 1) filtering request by namespace/name, 2) fetching antrea-install CR, 3) applying configuration, 4) setting not degraded. So how about using a single function to avoid duplicate codes?

Reply:
> For `func (k8s *AdaptorK8s) Reconcile` and `func (oc *AdaptorOc) Reconcile`, there are similar logic for 1) filtering request by namespace/name, 2) fetching antrea-install CR, 3) applying configuration, 4) setting not degraded. So how about using a single function to avoid duplicate codes?

ok, I'd like to try extract these logic to a common function.